BUILD

```diff
@@ -15,6 +15,17 @@ config_setting(
     },
 )
 
+################################################################################
+# ZLIB configuration
+################################################################################
+
+config_setting(
```

Comment:
We no longer need this new config_setting, right?

Reply:
Right.